docs/installation.rst

```diff
@@ -203,7 +203,7 @@ for compiling on Windows as provided by `@hodoulp <https://github.com/hodoulp>`_
     set CMAKE_PATH=D:\OpenSource\3rdParty\cmake-3.12.2
     set GLUT_PATH=D:\OpenSource\3rdParty\freeglut-3.0.0-2
     set GLEW_PATH=D:\OpenSource\3rdParty\glew-1.9.0
-    set PYTHON_PATH=C:\Python27
+    set Python_PATH=C:\Python27
```

Comment:
I guess the intention there was the environment variable `PYTHONPATH` in the first place? I suppose we'd need Patrick's input since I don't deal with the Windows builds, but seems like a reasonable assumption

Reply:
I think the similarity to `PYTHONPATH` is a coincidence. If you look at how it's being used, it's just a placeholder for the Python root directory, whearas `PYTHONPATH` would be a `;` delimited list of paths. We can check with him though.

Reply:
@hodoulp was `%PYTHONPATH%` supposed to be `%PYTHON_PATH%` in this script? `PYTHON_PATH` is defined toward the top of the script as the Python root dir, but in the cmake command, `%PYTHONPATH%` is used instead to prefix the paths which should exist in Python root. I _think_ this is a typo, but we wanted to check with you first. Thanks!

Reply:
I think you're right, but I just pushed a commit to revert this, since I'm doing this so I can build things on my machine to update the docs anyway, we can lump all the doc updates later.